### PR TITLE
Ensign Content Update Part 1

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request
Aseprite_iXjD3rWEnR
Now, ensigns finally have actual original content. They have proper uniforms and closets, all based on the HOP's.
StrongDMM_L8YUtAdJWg

The bridge armory has also been expanded into two sections, with one that is still locked behind captain access.
StrongDMM_Xlulw70fYX

There's still a lot more potential for more content, including proper jackets and hats for ensigns. The overall aesthetic of command should be finalized before we proceed further.

## Why It's Good For The Game

Placeholder assets bad. Also making ensigns feel less like assistants who broke into the bridge and more like a proper officer is good for immersion.

## Changelog
```changelog Toriate
add: Ensigns now have proper lockers and uniforms.
add: Bridge armory has been overhauled
```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/objects/items/weapons/storage/pouches.dm [hunk 1]
```diff
@@ -44,3 +44,17 @@
 
 /obj/item/weapon/tool/sword
 	w_class = ITEM_SIZE_BULKY
+
+/obj/item/weapon/storage/pouch/fanny
+	name = "fannypack"
+	desc = "An economical solution for general purpose belt-located storage. Holds less than a large pouch."
```
Comment (Jamini) [inline zzzz_modular_occulus/code/game/objects/items/weapons/storage/pouches.dm:50]: If it holds less than a large pouch on your waist, it's worthless. Just saying.

**Review by Jamini — CHANGES_REQUESTED** [review]
I'm not a fan of the captain-access portion of the bridge armory being behind a windoor either. It should be a full door with a rwall to the side.

**Comment by Michiyamenotehifunana** [comment]
StrongDMM_BgILARc6Ws
Armory is now more secure, tech loot added to lockers.
StrongDMM_sNgHlgPUEp

File: zzzz_modular_occulus/code/game/objects/structures/crates_lockers/closets/secure/captain.dm [hunk 2]
```diff
@@ -43,3 +43,20 @@
...
+	req_access = list(access_heads)
+	icon = 'zzzz_modular_occulus/icons/obj/closet.dmi'
+	icon_state = "ensign"
+
+/obj/structure/closet/secure_closet/reinforced/ensign/populate_contents()
+	new /obj/item/clothing/under/rank/ensign(src)
+	new /obj/item/device/radio/headset/heads/hop/ensign(src)
+	new /obj/item/clothing/shoes/leather(src)
+	new /obj/item/weapon/storage/backpack/satchel/leather/withwallet(src)
+	new /obj/item/weapon/storage/pouch/fanny(src)
```
[the remark this answers is not in the document]
Comment (Michiyamenotehifunana) [reply zzzz_modular_occulus/code/game/objects/structures/crates_lockers/closets/secure/captain.dm]: Resolved

File: zzzz_modular_occulus/code/game/objects/structures/crates_lockers/closets/secure/captain.dm [hunk 3]
```diff
@@ -43,3 +43,20 @@
...
+	name = "\improper Ensign's locker"
+	req_access = list(access_heads)
+	icon = 'zzzz_modular_occulus/icons/obj/closet.dmi'
+	icon_state = "ensign"
+
+/obj/structure/closet/secure_closet/reinforced/ensign/populate_contents()
+	new /obj/item/clothing/under/rank/ensign(src)
+	new /obj/item/device/radio/headset/heads/hop/ensign(src)
+	new /obj/item/clothing/shoes/leather(src)
+	new /obj/item/weapon/storage/backpack/satchel/leather/withwallet(src)
```
[the remark this answers is not in the document]
Comment (Michiyamenotehifunana) [reply zzzz_modular_occulus/code/game/objects/structures/crates_lockers/closets/secure/captain.dm]: Resolved.

File: zzzz_modular_occulus/code/game/objects/items/weapons/storage/pouches.dm (hunk 1 above)
Comment (Michiyamenotehifunana) [reply zzzz_modular_occulus/code/game/objects/items/weapons/storage/pouches.dm:50]: Irrelevant.

**Review by Jamini — APPROVED** [review]

**Pull request merged**